week-3/reading-list/script.js

```diff
@@ -1,4 +1,23 @@
-// for the tests, do not modify this array of books
+function readingList(books) {
+  const readingList = document.getElementById("reading-list");
+
+  for (const bookItem of books) {
+    const list = document.createElement("li");
+
+    list.setAttribute("class", bookItem.alreadyRead ? "green" : "red");
+
+    const p = document.createElement("p");
+    p.innerText = `${bookItem.title} by ${bookItem.author}`;
+
+    readingList.appendChild(list);
+    list.appendChild(p);
+
+    const image = document.createElement("img");
+    image.src = bookItem.bookCoverImage;
+    list.appendChild(image);
```

Comment:
I have abolutely nothing but good things to say. This is a really simple and elegant solution :) I like the use of CSS to make it somewhat pretty as well

Reply:
Thanks! 🙏